Dockerfile.exec-centos7-reprozip.fslbuild-centos5

```diff
@@ -0,0 +1,126 @@
+# CentOS with FSL.
+# Build via: docker build -f ./Dockerfile.exec-centos7-reprozip.fslbuild-centos5 -t centos7-reprozip .
+# Command to start the container : docker run -it --rm --security-opt seccomp=unconfined -w <working-dir-path> -v <local-dir-path-to-be-mounted>:/data centos7-repro /bin/bash
+# Option --security-opt seccomp=unconfined is in order to allow reprozip trace to work and trace the processes 
+# Command to run the prefreesurfer script : reprozip trace.txt PreFreeSurferPipelineBatch.sh --StudyFolder=$PWD --Subjlist=100408 --runlocal
+# --StudyFolder - The folder containing the subjects
+# --Subjlist - quoted, space separated list of subject IDs on which to run the pipeline
+# --runlocal     - if specified (without an argument), processing is run on "this" machine as opposed to being submitted to a computing grid
+FROM bigdatalabteam/hcp-prefreesurfer:exec-centos7-fslbuild-centos5-latest 
+MAINTAINER laletscaria@yahoo.co.in
+
+# Update the image with the latest packages
+# Note: the versions of these packages may change over time. 
+# Different builds of the container may contain different versions.
+
+############
+# Packages #
+###########
+
+# bc- calculator for fslreorient2std
+# curl-devel - cURL is a software package which consists of command line tool and a library for transferring data using URL syntax
+# epel-release to help pip installation
+# expat-devel - xml parser library
+# fontconfig - Fontconfig package contains a library and support programs used for configuring and customizing font access
+# freetype contains a library which allows applications to properly render TrueType fonts
+# gcc -  compiler system produced by the GNU Project supporting various programming languages like c,c++,java etc.
+# gettext helps in producing multi-lingual messages
+# git library for cloining pipelines repository.
+# libpng - libpng is the official PNG reference library
+# libSM - is a library of generally useful C abstractions
+# libstdc++ - a powerful set of reuseable tools in the form of the C++ Standard Library
+# libXrender - used when compiling programs that use freetype fonts
+# libXext - Xorg libraries provide library routines that are used within all X Window applications
+# openssl-devel - for secure communication
+# python and python-devel - To support reprozip trace
+# perl-ExtUtils-MakeMaker - designed to write a Makefile for an extension module from a Makefile.PL
+# sqlite-devel - package contains the header files and development documentation for sqlite
+# tar,unzip - to extract .tar.gz and .zip files
+# wget to get files or status from URL's
+# zlib-devel - to support FMRIB file IO
+# numpy - to support FMRIB pipeline execution
```

Comment:
why are the package installation lines commented? Just remove them if they aren't used.